crates/proto/src/h3/h3_client_stream.rs

```diff
@@ -321,7 +328,8 @@ impl H3ClientStreamBuilder {
         server_name: Arc<str>,
         path: Arc<str>,
     ) -> H3ClientConnect {
-        H3ClientConnect(Box::pin(self.connect(name_server, server_name, path)) as _)
+        let disable_grease = self.disable_grease;
+        H3ClientConnect(Box::pin(self.connect(name_server, server_name, path, disable_grease)) as _)
```

Comment:
nit: `self.disable_grease` doesn't need to be an argument here, since it's already available via `self`.